examples/vision/ipynb/supervised-contrastive-learning.ipynb

```diff
@@ -24,7 +24,7 @@
     "\n",
     "[Supervised Contrastive Learning](https://arxiv.org/abs/2004.11362)\n",
     "(Prannay Khosla et al.) is a training methodology that outperforms\n",
-    "plain crossentropy-supervised training on classification tasks.\n",
+    "supervised training on classification tasks with crossentropy.\n",
```

Comment:
This should be "supervised training with crossentropy on classification tasks"
or "crossentropy-supervised training on classification tasks"

note that the original version was fine!

Reply:
Done!